dmutils/forms/fields.py

```diff
@@ -28,3 +35,83 @@ def __init__(self, label=None, **kwargs):
             ),
         ))
         super(EmailField, self).__init__(label, **kwargs)
+
+
+class DateField(Field):
+    '''
+    A date field(set) that uses a day, month and year field.
+
+    It behaves like a WTForms.FieldForm, but it can be used
```

Comment:
Do you mean `FormField` here instead of `FieldForm`?

Reply:
Woops, good spot